### ISSUE TITLE
3.2.2-0: Cannot decrypt data

### ISSUE THREAD

**Issue by alex-robbins** [TeamWin/android_device_oneplus_dumpling#3]
With TWRP 3.2.2-0, when I boot TWRP and type in my encryption password, TWRP cannot decrypt the data partition. The log says 
```
E: This TWRP does not have synthetic password decrypt support
Unable to decrypt with default password. You may need to perform a Format Data.
```

I did not have this problem with version 3.2.1-2.

**Comment by alex-robbins** [comment]
Others have mentioned this problem [on Reddit], so it isn't just me.

[on Reddit]: https://www.reddit.com/r/Android/comments/8w1lgq/twrp_3220_is_available_now/e1sml6z/

**Comment by alex-robbins** [comment]
This appears to be fixed in 3.2.3-1.
